src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -258,7 +258,8 @@ extern bool entitiesQuery
   const std::string&               sortOrderList  = "",
   bool                             includeCreDate = false,
   bool                             includeModDate = false,
-  const std::string&               apiVersion     = "v1"
+  const std::string&               apiVersion     = "v1",
+  StringFilter*                    stringFilterP  = NULL
```

Comment:
If we move StringFilter to Scope, we don't need this new paramter (it will come "inside" `const Restriction& res`).